src/Scrapers/Scraper.py

```diff
@@ -8,6 +12,15 @@ class Scraper:
     def __init__(self, url) -> None:
         self.url = url
 
+    def get_response(self):
```

Comment:
Since this is a private method, put '__' before the name according to the naming convention.

Reply:
Have made this function private now.